README.md

```diff
@@ -34,51 +34,72 @@ In order to see the nice font as we used in the demo, add the following rule in
 ```
 
 ## Workshop steps
-- step 0: install Meteor and test locally
+- step 0:
+  - install Meteor and test locally (TODO: link toevoegen naar meteor of anders de stappen)
+  - follow the instructions and run it.
+  - change some things in the code and see what happens
 - step 1: setup the Tic tac toe project
+  - copy the content of the css in the css file generated in your folder
+  - Notice that you don't have to update your browser. Meteor does that for you automagically
   - you can rename the template in your `.html` file, for example `<template name="gameboard"></template>` instead of the `hello` template
-  - begin with `<div class="gameboard"></div>` (Now you should see a nice green schoolboard)
-  - you can add `<div class="box"></div>` into the `.gameboard` in your template
+  - begin with `<div class="gameboard"></div>` in this template(Now you should see a nice green schoolboard)
+  - you can add `<div class="box"></div>` into the `gameboard` div
+  - Tip: with F12 you can open the developers tools of your browser. And if you use console.log (TODO: Link to console.log) in the js file you can print to the console.
 - step 2: display the grid (3 x 3)
-    - you can use for example, [Mongo Collection](http://docs.meteor.com/#/full/mongo_collection)
-    - you can follow the steps explained in [Meteor tutorial](https://www.meteor.com/tutorials/blaze/templates). Instead of tasks from the tutorial, we want to make boxes
-    - you probably want to use a loop in order to create a scalable application, like this:
-    ```javascript
-    for(var i = 0; i < CellsCount; i++){}
+  - there is only one box on our gameboard. For tictactoe we need 9 of them. But we don't want to write 9 divs in our code.
+  - you probably want to use a loop in order to create a scalable application, like this:
+  ```javascript
+  for(var i = 0; i < CellsCount; i++){}
     ```
-    - You would want to update the collection only once
-    - make sure you register this collection in the server part of the code. Can you predict / test what happens if you register the collection on the client side?
-    - use css classname `.box` for each cell to take advantage of the provided styling (tictactoe.css)
+  - The js code is not doing a lot. Server and Client part(TODO: Link to it) The most part we will do today will be on the client.
+  - There are event and helper function for every template that is made in the html. The hello template is not there anymore. Change the functions so that the new templates are used.
```

Comment:
helpers is an object to register helper functions. Remove events here: only relevant in step 3